### ISSUE TITLE
irfaq to in2faq CLI command broken

### ISSUE THREAD

**Issue by bmack** [in2code-de/in2faq#3]
Hey in2code team,

I wanted to try out the extension for TYPO3 v11 and migrate from irfaq to in2faq. I installed the extension but it seems like there is a migration to "tx_in2faq_domain_model_expert" which I can't find in ext_tables.sql. Can you shed some light on the upgrader? Can this safely be removed? If so, I can prepare a PR.

Thanks in advance for a quick info.
Benni.

**Comment by einpraegsam** [comment]
Hi Benni,

this extension was mainly used for internal usage for some of our customers. So we had no time for a quality check for the public usage. That's why we did not publish it in TER yet. E.g. there is no JavaScript delivered, because our sitepackage extensions mostly delivers some code for accordeon functionality - we simply adjust the HTML templates of in2faq.

We once used it to migrate from irfaq to a newer model. Because this was already many years ago and because we did not use the migrator again since, I'm thinking of dropping this functionality. In2faq does not support any experts. This part should be removed from the migration at all.

If you still want to use in2faq (the little TCA and HTML-Templates could also be done by you in your sitepackage or an additional extension), PR are welcome :)

Greetings from Rosenheim, Alex
